code/game/objects/items/weapons/teleprod.dm

```diff
@@ -1,9 +1,11 @@
 /obj/item/weapon/melee/baton/cattleprod/teleprod
 	name = "teleprod"
 	desc = "A prod with a bluespace crystal on the end. The crystal doesn't look too fun to touch."
+	w_class = 3
```

Comment:
because i don't have sprites for it and i don't think i've ever seen a teleprod being used